src/main/java/uk/gov/companieshouse/bankruptofficersearch/api/exception/GlobalExceptionHandler.java

```diff
@@ -29,10 +29,10 @@
     public ResponseEntity<Object> handleOracleQueryApiException(OracleQueryApiException ex) {
 
         HashMap<String, Object> message = new HashMap<>();
-        message.put("message", ex.getStatusText());
+        message.put("message", ex.getMessage());
         message.put("error", ex.getClass());
-        message.put("statusCode", ex.getStatusCode());
-        LOGGER.error(ex.getMessage(), message);
+        message.put("stack_trace", ex.getStackTrace());
+        LOGGER.error(ex, message);
```

Comment:
The `message` and `stack_trace` fields are populated [here](https://github.com/companieshouse/structured-logging/blob/ac39413c10d3bff38dbf923e29dc0981e9ff35b9/src/main/java/uk/gov/companieshouse/logging/StructuredLogger.java#L230) so doesn't need to be done in this handler.

```
@ExceptionHandler(OracleQueryApiException.class)
public ResponseEntity<Object> handleOracleQueryApiException(OracleQueryApiException ex) {
    LOGGER.error(ex);
    return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
}
```

Reply:
Removed unnecessary fields